gensim/models/atmodel.py

```diff
@@ -882,6 +882,84 @@ def get_document_topics(self, word_id, minimum_probability=None):
 
         raise NotImplementedError('Method "get_document_topics" is not valid for the author-topic model. Use the "get_author_topics" method.')
 
+    def get_new_author_topics(self, corpus, minimum_probability=None):
```

Comment:
What's a reason to `minimum_probability=None` here (instead of 1e-8)?

Reply:
@menshikh-iv It is the same in `get_document_topics` in LDA (https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/ldamodel.py#L987), and `get_term_topics` as well.

Reply:
so, I should leave it like this, right?

Reply:
@Stamenov yes, leave as is